### PR TITLE
togglePaneCollapse update

### PR DESCRIPTION
**Pull request by tohmjudson**
Associated Issue: #4482 

### Summary of Changes
Refactor togglePaneCollapse check before dispatch

### Screenshots/Videos
nrwr3ti3zt

### CODE REVIEW

File: src/actions/ui.js [hunk 1]
```diff
@@ -78,10 +78,17 @@ export function showSource(sourceId: string) {
...
+    const getPaneCollapseState = getPaneCollapse(getState(), position);
+    if (getPaneCollapseState === paneCollapsed) {
+      return;
+    }
+
+    dispatch({
+      type: "TOGGLE_PANE",
+      position,
+      paneCollapsed
+    });
```
Comment (jasonLaster) [inline src/actions/ui.js]: this is a good opportunity to update the tests in `actions/test/ui.spec.js`

File: src/actions/ui.js [hunk 2]
```diff
@@ -78,10 +78,17 @@ export function showSource(sourceId: string) {
 }
 
 export function togglePaneCollapse(position: string, paneCollapsed: boolean) {
-  return {
-    type: "TOGGLE_PANE",
-    position,
-    paneCollapsed
+  return ({ dispatch, getState }: ThunkArgs) => {
+    const getPaneCollapseState = getPaneCollapse(getState(), position);
```
Comment (wldcordeiro) [inline src/actions/ui.js]: Maybe `prevPaneCollapseState`?

Comment (tohmjudson) [reply src/actions/ui.js]: I like that much better. Thanks!

File: src/actions/ui.js (hunk 1 above)
Comment (tohmjudson) [reply src/actions/ui.js]: Sounds like a good idea, I'll take a look

**Pull request merged**